src/ColumnSchema.php

```diff
@@ -87,7 +86,7 @@ public function dbTypecast(mixed $value): mixed
 
             Schema::TYPE_BIT => is_int($value)
                 ? str_pad(decbin($value), (int) $this->getSize(), '0', STR_PAD_LEFT)
-                : $this->typecast($value),
```

Comment:
I'm not sure, but if `$value`  should be returns as string, then php type [here](https://github.com/yiisoft/db-pgsql/blob/7711050e1cfad9d9ebbd7fbda5ff3600dd345a67/src/Schema.php#L831) also must be string instead of integer. And then not need remove typecast here.

Reply:
For PHP it is `int` for DB it is `string`.
Which type the property `phpType` show?
If "for PHP" then it should be `int`. In this case `$this->typecast($value)` will return wrong value for DB.
I think there might be cases when raw values (retrieving from the DB) would be passed in `dbTypecast()` and the raw value for `bit` type will be `string`.

It should return `$value` or `(string) $value` instead of `$this->typecast($value)`.

Reply:
In that case your suggestion `(string) $value` is good.